taccsite_cms/templates/snippets/mailto-link.css.html

```diff
@@ -0,0 +1 @@
+<link rel="stylesheet" href="/static/site_cms/css/build/mailto-link.css" />
```

Comment:
Any template inside `templates/snippets` is available to be loaded by a snippet, thus I can easily reuse immutable DRY bits of code on the CMS where needed.